third_party/maven_install.json

```diff
@@ -1,13 +1,14 @@
 {
     "dependency_tree": {
         "__AUTOGENERATED_FILE_DO_NOT_MODIFY_THIS_FILE_MANUALLY": "THERE_IS_NO_DATA_ONLY_ZUUL",
-        "__INPUT_ARTIFACTS_HASH": -1770608701,
+        "__INPUT_ARTIFACTS_HASH": 1212292329,
         "__RESOLVED_ARTIFACTS_HASH": -348454682,
         "conflict_resolution": {
             "androidx.appcompat:appcompat:1.0.2": "androidx.appcompat:appcompat:1.2.0",
             "androidx.core:core:1.0.1": "androidx.core:core:1.3.0",
             "androidx.recyclerview:recyclerview:1.0.0": "androidx.recyclerview:recyclerview:1.1.0",
             "androidx.test:core:1.0.0": "androidx.test:core:1.2.0",
+            "com.squareup.okhttp3:okhttp:3.12.12": "com.squareup.okhttp3:okhttp:4.1.0",
```

Comment:
@BenHenning After downgrading the deps I ran the 
`REPIN=1 bazel run @unpinned_maven//:pin` which was successful but now its showing 2 versions of same library.
I think this might be because of `Retrofit` library.

Reply:
This seems really problematic. We need to be able to update to newer versions otherwise we might lose out on security patches.

Did OkHttp officially drop support for pre-SDK 21?

Reply:
https://github.com/square/okhttp/issues/4481 paints a pretty clear picture here. We might need to find a different HTTP library to use instead of OkHttp since their <21 compatibility store isn't great. Retrofit will probably force us to use a newer version of OkHttp.

I'm not sure on next steps yet. Retrofit seems to only support OkHttp, and we would either need to backport Retrofit support (limiting our update choices) or find an alternative. For alternatives, I've looked into gRPC (which would be better long-term, anyway), but App Engine doesn't currently support gRPC (or rather, its load balancer will block gRPC traffic).

We could potentially compile out the data module to remove the dependency entirely but that requires shipping the app with Bazel and we can't do that until the app bundles portion of #2432 is addressed. I think that's currently in progress, but probably won't be ready for a few more weeks. We could look into manually building the AAB temporarily, but per Android developer documentation that's probably going to be difficult with databinding (can't be sure until we try, but it requires manually running aapt2 to build out resources).

Finally, we could manually force the data module out of the app via Gradle but that may result in a potential risk to regress existing tests that expect everything to hook-up correctly.

I actually think our hands might be tied here--I don't see an obvious way forward that may not result in a large time commitment to make this work.

Reply:
Update: Per https://github.com/oppia/oppia-android/issues/2432#issuecomment-905111680 if we can build AABs via Bazel then we can isolate the data module for SDK 19 devices. It doesn't solve the long-term solution, but it might be a viable workaround to at least unblock the immediate need to have SDK 19 support in Alpha MR3.